app/services/order_transfer.rb

```diff
@@ -0,0 +1,30 @@
+class OrderTransfer < Rectify::Command
+  def initialize(user:, order_id:)
+    @user = user
+    @order_id = order_id
```

Comment:
Remove this variable and add to initializer `@order = Order.find_by_id(order_id)`.
Also please use `attr_reader` for `@user` and `@order`